eng/pipelines/update-azure-linux-pipeline.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For info about runtime parameters, see https://github.com/microsoft/go-infra/blob/main/docs/pipeline-yml-style.md#runtime-parameters
+parameters:
+  # If someone's manually queueing this build, let them skip the approval stage. It takes time to
+  # acquire a serverless agent to present the approval step, so the dev has to pay attention and
+  # catch it. Instead, let them approve it ahead of time by skipping it. The default is always used
+  # by automation, so this is only used for dev builds and retries.
+  - name: approveAheadOfTime
+    displayName: Approve right now, skipping the approval stage.
+    type: boolean
+    default: false
+
+  # Allow disabling specific parts of the internal build release. These can be used to re-run
+  # specific parts of a release if necessary, e.g. if a step silently failed.
+  - name: runCreatePullRequest
+    displayName: Create Pull Request on Azure Linux repo
+    type: boolean
+    default: true
+
+  # This parameter intentionally has no default: see release-go-pipeline.yml
+  - name: goReleaseConfigVariableGroup
+    displayName: '[Use "go-release-config" for a real release] Variable group that specifies release target locations and auth.'
+    type: string
+
+trigger: none
+pr: none
+
+variables:
+  - group: Microsoft-GoLang-bot
+  # Import config group. This may direct the build to use secrets from the other groups.
+  - group: ${{ parameters.goReleaseConfigVariableGroup }}
+  - name: assetJsonPath
+    value: $(Pipeline.Workspace)/build/BuildAssets/assets.json
```

Comment:
What's this variable used for?

There's already a variable in the releasego.yml template that I would expect to serve this purpose:

https://github.com/microsoft/go-infra/blob/f0ee8dc8eb530988cbb8a971ce9290b8a9484704/eng/pipelines/jobs/releasego.yml#L37-L40

Reply:
I think Mert added it for testing, looks like we can remove it and rely on `buildAssetJsonFile`

Reply:
fixed